torch/ao/quantization/pt2e/representation/rewrite.py

```diff
@@ -22,6 +23,35 @@
     torch.tensor([127], dtype=torch.int),
 )
 
+def _qdq_quantized_add_relu(x_i8, x_scale, x_zero_point, y_i8, y_scale, y_zero_point, out_scale, out_zero_point, quant_min, quant_max):
+    x_fp32 = torch.ops.quantized_decomposed.dequantize_per_tensor(x_i8, x_scale, x_zero_point, quant_min, quant_max, torch.int8)
+    y_fp32 = torch.ops.quantized_decomposed.dequantize_per_tensor(y_i8, y_scale, y_zero_point, quant_min, quant_max, torch.int8)
+    out_fp32 = x_fp32 + y_fp32
+    out_fp32 = torch.ops.aten.relu(out_fp32)
+    out_i8 = torch.ops.quantized_decomposed.quantize_per_tensor(
+        out_fp32, out_scale, out_zero_point, quant_min, quant_max, torch.int8
+    )
+    return out_i8
+
+def _reference_quantized_add_relu(
+    x_i8, x_scale, x_zero_point, y_i8, y_scale, y_zero_point,
+    out_scale, out_zero_point, quant_min, quant_max
+):
+    """
+    # See comments for `_reference_quantized_add` for more information on how to derive the formula for out_i8 based on x_i8 and y_i8
+    """
+    x_i32 = x_i8.to(torch.int32)
+    y_i32 = y_i8.to(torch.int32)
+    # TODO: change this to mul.Scalar?
+    x_i32 = out_dtype(torch.ops.aten.mul.Tensor, torch.int32, (x_i32 - x_zero_point), (x_scale / out_scale))
+    y_i32 = out_dtype(torch.ops.aten.mul.Tensor, torch.int32, (y_i32 - y_zero_point), (y_scale / out_scale))
+    out_i32 = x_i32 + y_i32 + out_zero_point
+    out_i32 = torch.ops.aten.clamp(out_i32, out_zero_point)
+    quant_min = -128
```

Comment:
quite strange to accept the `quant_min` as a method argument and then reset it like this. can this hide some bugs (if the passed `quant_min` is different from `-128`)?

Reply:
sorry, will update qmin/qmax and make it dtype agnostic as well. for add representation, it is not updated to use out_dtype yet, so I'll probably have a separate PR for that

Reply:
for dtype, can I do it a bit later to reduce the scope of change? it should be relatively straightforward

Reply:
> for dtype, can I do it a bit later to reduce the scope of change? it should be relatively straightforward

Is this related to my comment? If it is straightforward and easy to include in this diff, please do so. Otherwise go ahead.

Reply:
this is related to some helper functions we added later in the stack (the replace literal with placeholder function) so I plan to have a separate PR after all these are done to include this support